web/app/Http/Middleware/TrustHosts.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Wikijump\Http\Middleware;
+
+use Illuminate\Http\Middleware\TrustHosts as Middleware;
+
+class TrustHosts extends Middleware
+{
+    /**
+     * Get the host patterns that should be trusted.
+     *
+     * @return array
+     */
+    public function hosts()
+    {
+        return [
+            $this->allSubdomainsOfApplicationUrl(),
```

Comment:
Is this correct? If `allSubdomainsOfApplicationUrl()` returns a list it'll be returning `[ [ domain... ] ]`

Reply:
`allSubdomainsOfApplicationUrl()` returns a string or null.